service/api/types.go

```diff
@@ -2,45 +2,72 @@ package api
 
 // DebuggerState represents the current context of the debugger.
 type DebuggerState struct {
-	// BreakPoint is the current breakpoint at which the debugged process is
-	// suspended, and may be empty if the process is not suspended.
+	// BreakPoint is the current breakpoint at which the debugged process
+	// is suspended, and may be empty if the process is not suspended.
 	BreakPoint *BreakPoint `json:"breakPoint,omitempty"`
 	// CurrentThread is the currently selected debugger thread.
 	CurrentThread *Thread `json:"currentThread,omitempty"`
+	// List is the current focus for printing source lines.
+	List *List `json:"list,omitempty"`
 	// Exited indicates whether the debugged process has exited.
 	Exited bool `json:"exited"`
 }
 
+// Arguments holds the unparsed command line argument(s).
+type Arguments struct {
```

Comment:
This name seems to general, and I don't think it buys us anything over just using a slice.